Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_IOCs_-_Push_new_IOCs_to_XDR_Main.yml

```diff
@@ -0,0 +1,274 @@
+id: Cortex XDR IOCs - Push new IOCs to XDR (Main)
+version: -1
+name: Cortex XDR IOCs - Push new IOCs to XDR (Main)
+description: This is the *main* playbook for XDR IOCs sync. The playbook will "sync" IOCs into XDR by pushing new IOCs in and disabling expired IOCs. The playbook utilizes XSOAR tags and loops in order to find IOCs using a query provided by the user. The play book will iterate over the IOCs pushing them in batches into XDR. In the second phase the playbook will disable expired  IOCs that were previously pushed into XDR. We recommend running this playbook as a Job a few times a day after disabling the integration sync function.
```

Comment:
I think running this playbook once a day will be enough, dont you?

Reply:
At least twice I would say, what do you think?